pylint/checkers/logging.py

```diff
@@ -238,20 +249,36 @@ def _check_log_method(self, node, name):
                 )
                 emit = total_number_of_strings > 0
             if emit:
-                self.add_message("logging-not-lazy", node=node)
+                self.add_message(
+                    "logging-not-lazy", node=node, args=(self._helper_string(node),),
+                )
         elif isinstance(node.args[format_pos], astroid.Call):
             self._check_call_func(node.args[format_pos])
         elif isinstance(node.args[format_pos], astroid.Const):
             self._check_format_string(node, format_pos)
-        elif isinstance(
-            node.args[format_pos], (astroid.FormattedValue, astroid.JoinedStr)
+        elif isinstance(node.args[format_pos], astroid.JoinedStr):
+            self.add_message(
+                "logging-fstring-interpolation",
+                node=node,
+                args=(self._helper_string(node),),
+            )
+
+    def _helper_string(self, node):
+        """Create a string that lists the valid types of formatting for this node."""
+        valid_types = ["lazy %"]
```

Comment:
Is it possible to have `lazy {}` when `logging-format-style=new`?
Or something similar that doesn't lead to old-style (`%`)?